### ISSUE TITLE
What do the numbers mean?

### ISSUE THREAD

**Issue by jgentil** [tortoise/orm-benchmarks#4]
The scale and units are entirely missing, both from this data, the documentation around it, and it's use in an image on the tortoise README.rst file.   Are they operations per second?  On what kinda machine specs?  Perhaps adding a relatively known non-ORM computation in the same scale and units would be beneficial to see how well the test machine compares to someone else's.

**Comment by rohfle** [comment]
Looks like the units are in Rows/sec:
https://github.com/tortoise/orm-benchmarks/blob/master/src/pony/test_g.py#L20
```print(f'Pony ORM, G: Rows/sec: {count / (now - start): 10.2f}')```

**Comment by daveflr** [comment]
Yeah but I have the same question, more is the better? lower is the better? seems that info is missing in the readme file

**Comment by staniec** [comment]
I also ended up checking the code upon the same confusion.
